pytest/test_register.py

```diff
@@ -1993,3 +1993,40 @@ def testStreamReaderUnregisterFinishesOnlyCurrentBatch(env):
                 time.sleep(0.1)
     except Exception as e:
         pass
+
+
+@gearsTest(skipOnCluster=True)
+def testFastPauseAndUnpause(env):
+    script = '''
+import time
+regId = GB('StreamReader').foreach(lambda x: execute('incr', 'x')).foreach(lambda x: time.sleep(1)).register()
```

Comment:
python variables should be slit with underscores (reg_id) or regid. Not camel cased